AlwaysOn/Wearable/src/main/java/com/example/android/wearable/wear/alwayson/MainActivity.kt

```diff
@@ -0,0 +1,366 @@
+/*
+ * Copyright (C) 2015 Google Inc. All Rights Reserved.
```

Comment:
Minor nit, you should update the copyrights to 2021.

Reply:
Updated/added source headers for everything in `AlwaysOn`